discovery.md

```diff
@@ -215,19 +214,29 @@ The following sections define the attributes that appear in a Service entity.
   specification makes no statement, or guarantees, as to the forwards or
   backwards compatibility of a Service as it changes over time.
 
-  Note, unlike the `name` attribute which only needs to be unique within the
-  scope of a single Discovery Endpoint, the global uniqueness aspect of this
-  attribute allows for the discovery of the same Service across multiple
-  Discovery Endpoints.
-
   See the Primer for more information.
 
 - Constraints:
   - REQUIRED in responses from the Discovery Endpoint.
-  - MUST be a valid UUID per RFC4122.
 - Examples:
   - `bf5ff5cc-d059-4c79-a89a-2513e45a1340`
 
+#### authority (service authority)
+
+- Type: `URI`
+- Description: Identifies the authority for this service. Similar to schemas authority.
+  Note, the pair `authority` and `id` are globally unique
+- Constraints:
+  - OPTIONAL. If the attribute is absent or empty, its implied default value is the base
+    URI of the API endpoint.
+  - MUST be a valid URI.
+  - For services imported from other discovery endpoints in replication scenarios, the
+    attribute is REQUIRED to be not empty. If the value is empty or absent
+    during import, it MUST be explicitly set to its implied default value.
+- Examples:
+  - `urn:com-example-schemas`
+  - `https://schemas.example.com`
+
 ##### epoch
```

Comment:
If we can agree on the concept of `authority`, we may revisit the definition of `epoch`. I think it should only be increased by the DE that represents the authority and then be used for replicating discovery data downstream.

Reply:
Does this mean that only the authority can edit it? Even to make local changes like update the URLs? Or does that mean it needs to be a new ID at that point and lose it's history/linkage?